frontend/src/app/static/faq-organization.component.html

```diff
@@ -0,0 +1,98 @@
+<div class="container">
```

Comment:
I know, that it's mostly copied from `backend/apps/volontulo/templates/pages/faq-organizations.html` (BTW it should be removed in this pull request), but we decided to go with 2-space indentations in html (in old html files in this repo You can find 4-spaces, but that's depraceted).